recipes/itk/all/conanfile.py

```diff
@@ -0,0 +1,288 @@
+import os
+import re
+import shutil
+
+from conans import ConanFile, CMake, tools
+from fnmatch import fnmatch
+
+
+class LibITKConan(ConanFile):
+    name = "itk"
+    upstream_version = "4.13"
+    upstream_patch = "2"
+    version = "{0}.{1}".format(upstream_version, upstream_patch)
+
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False]}
+
+    default_options = {
+        "shared": True,
+        "boost:shared": True,
+        "zlib:shared": False}
+
+    exports = [
+        "patches/CMakeProjectWrapper.txt",
+        "patches/linux.diff"
+    ]
+    homepage = "http://www.itk.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "http://www.itk.org/licensing/"
+    description = "Insight Segmentation and Registration Toolkit"
+    source_subfolder = "sf"
+    build_subfolder = "build"
+    short_paths = True
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        if 'CI' not in os.environ:
+            os.environ["CONAN_SYSREQUIRES_MODE"] = "verify"
+
+    def requirements(self):
+
+        if not tools.os_info.is_linux:
+            self.requires("libjpeg/9c@bincrafters/stable")
+            self.requires("libpng/1.6.37@bincrafters/stable")
+            self.requires("libtiff/4.0.9@bincrafters/stable")
+
+        if tools.os_info.is_windows:
+            self.requires("Expat/2.2.6@pix4d/stable")
+            self.requires("zlib/1.2.11@conan/stable")
+
+    def build_requirements(self):
+        if tools.os_info.linux_distro == "linuxmint":
+            pack_names = [
+                "libexpat1-dev",
+                "libjpeg-turbo8-dev",
+                "libtiff5-dev"
+            ]
+            if tools.os_info.os_version.major(fill=False) == "18":
+                pack_names.append("libpng12-dev")
+            elif tools.os_info.os_version.major(fill=False) == "19":
+                pack_names.append("libpng-dev")
+            installer = tools.SystemPackageTool()
+            for p in pack_names:
+                installer.install(p)
+
+    def system_requirements(self):
+        if tools.os_info.linux_distro == "linuxmint":
+            pack_names = [
+                "libexpat1",
```

Comment:
We do have an `expat` recipe, please use this instead

Reply:
done